app/views/restaurants/show.html.erb

```diff
@@ -14,9 +14,9 @@
     <div>Type of Cuisine:</div>
     <% if @restaurant.cuisine.present? %>
```

Comment:
If you choose to use `<%-something-%>`, you'd want that to be reflected here too. It's not just `<%-end-%>` syntax, but all erb tags with whitespace.